ocs_ci/utility/utils.py

```diff
@@ -2356,16 +2435,35 @@ def run_ceph_health_cmd(namespace):
 
     """
     # Import here to avoid circular loop
-    from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+    if namespace == constants.OPENSHIFT_STORAGE_EXTENDED_NAMESPACE:
+        from ocs_ci.utility.connection import Connection
+        from ocs_ci.deployment.helpers.external_cluster_helpers import (
+            get_external_cluster_client,
+        )
 
-    try:
-        ct_pod = get_ceph_tools_pod(namespace=namespace)
-    except (AssertionError, CephToolBoxNotFoundException) as ex:
-        raise CommandFailed(ex)
+        host, user, password, ssh_key = get_external_cluster_client()
+        if not password and ssh_key:
+            connection_to_cephcluster = Connection(
+                host=host, user=user, private_key=ssh_key
+            )
+        elif password:
+            connection_to_cephcluster = Connection(
+                host=host, user=user, password=password
+            )
+        ceph_health_tuple = connection_to_cephcluster.exec_cmd("ceph health")
+        return ceph_health_tuple[1]
 
-    return ct_pod.exec_ceph_cmd(
-        ceph_cmd="ceph health", format=None, out_yaml_format=False, timeout=120
-    )
+    else:
+        from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+
+        try:
+            ct_pod = get_ceph_tools_pod(namespace=namespace)
+        except (AssertionError, CephToolBoxNotFoundException) as ex:
+            raise CommandFailed(ex)
```

Comment:
why is try except needed here? Simply let the original exception to be raised

Reply:
If it's taken from other place I agree to keep it this way as it was probably like that for some reason and changing it brings risk of other regression.